.devcontainer/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # See here for image contents: https://github.com/devcontainers/images/blob/main/src/dotnet/.devcontainer/Dockerfile
 
 # [Choice] .NET version: 6.0, 3.1, 6.0-bullseye, 3.1-bullseye, 6.0-focal, 3.1-focal
```

Comment:
Should update comment here.

Reply:
I should have seen that the first time 😅